server/auto_updater/embedding_bin_updater.py

```diff
@@ -43,6 +43,7 @@ def generate_centroid_dataset(paper_dataset_file, output_file="centroid_dataset.
         if line["journal"] not in journal_centroid:
             journal_centroid[line["journal"]] = {
                 "journal": line["journal"],
+                "document": line["document"],
```

Comment:
@danich1 So this means the representative paper in a journal will be always the first paper in `dataset_dataset_full.tsv`?

Reply:
Sorry was in a car all day. 
Yeah. That's what i originally had as a representative paper.